package.json

```diff
@@ -56,7 +56,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "yarn prettier && yarn lint-contracts",
-      "pre-push": "yarn test-js"
+      "pre-push": "yarn tests-js"
```

Comment:
I believe it is `test-js` only, as it calls the `test-js` script in Line 52. Which in turn calls two main ones:
- hardhat (which is configured in its own config file)
- ganache (which is also configured differently for some tests which are not yet migrated to hardhat)